src/open_inwoner/openzaak/cases.py

```diff
@@ -102,30 +92,28 @@ def add_status_type_config(case: Zaak) -> None:
         pass
 
 
-def preprocess_data(cases: list[Zaak]) -> list[Zaak]:
+def preprocess_data(cases: list[Zaak], group: ZGWApiGroupConfig) -> list[Zaak]:
     """
     Resolve zaaktype and statustype, add status type config, filter for visibility
 
     Note: we need to iterate twice over `cases` because the `zaak_type` must be
           resolved to a `ZaakType` object before we can filter by visibility
     """
```

Comment:
The docstring is out of date, now that `preprocess_case` is inside this function (though it's not the present commit). We're iterating several times over `cases`, not just twice.

Reply:
You're right, got it on second look.